asq-exercise-progress.html

```diff
@@ -0,0 +1,612 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../asq-base/asq-base.html">
+<link rel="import" href="asq-exercise-progress-style.html">
+
+<!--
```

Comment:
Add some documentation on what this element does, what information it shows and maybe some design decisions. For example

> asq-exercise-progress is used to display the viewers' progress in answering an exercise. It consists of 4 partial progress bars: one that displays the total number of viewers that have submitted, one for ...